src/lib/actionrunner.js

```diff
@@ -169,18 +169,21 @@ class OpenwhiskActionRunner {
                 ${mounts}
                 ${this._getImage()}`
         );
+        
+        const rawHost = await this._docker(`port ${this.containerId} ${RUNTIME_PORT}`);
+        this.containerHost = `0.0.0.0:${rawHost.slice(-5)}`;
```

Comment:
only issue is if ip address is 4 instead of 5 digits. I should pop off everything after colon instead of slicing a certain amount